### PR TITLE
Fix base.html URLs

### PR DESCRIPTION
**Pull request by Mahhheshh**
Solves #371 
Updated the URLs used in the footer of the application to use the correct links. Previously, the links were pointing to the wrong resources, but have been updated to point to the correct URLs. This includes updating the links to the AiiDAlab registry page in the aiidalab-registry repository. These changes ensure that users can access the correct resources from the footer of the application.

This pull request fixes an issue in the parent file `base.html` at https://github.com/aiidalab/aiidalab/blob/50498b08ea77937cb8af0e8d753446dd17b6d637/aiidalab/registry/templates/base.html#L3 which was causing issues in the child file `layout.html` at https://github.com/aiidalab/aiidalab-registry/blob/8bc5d00fffa2016f562d065d81bd5a111c99479e/src/templates/layout.html#L8. The issue was causing variables to not be overwritten in the registry repository. The changes made in this pull request fix the links in the footer, and ensure that the variables can be correctly overwritten in the child file.

### CODE REVIEW

**Review by yakutovicha — CHANGES_REQUESTED** [review]
thanks, @Mahhheshh. Could you also fix the links in: `docs/source/conf.py` and `docs/source/usage/index.rst`? They still point to the old AiiDAlab URL.

**Comment by yakutovicha** [comment]
> Yes, are there any additional links I'm missing?

I think that is all.

File: docs/source/usage/index.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ Access AiiDAlab
 
 As a user, you have three options to access AiiDAlab:
 
- 1. Log into one of the `open AiiDAlab servers <https://materialscloud.org/aiidalab>`_.
+ 1. Log into one of the `open AiiDAlab servers <https://www.aiidalab.net/>`_.
```
Comment (yakutovicha) [inline docs/source/usage/index.rst:12]:
```suggestion
 1. Log into one of the `open AiiDAlab servers <https://www.aiidalab.net/deployments/>`_.
```

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
Comment (yakutovicha) [inline Pipfile:1]: I don't think this file is needed.

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Thanks, just a few small changes.

File: Pipfile (hunk 2 above)
Comment (Mahhheshh) [reply Pipfile:1]: Sorry for that I will remove it

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**